README.md

```diff
@@ -183,155 +157,12 @@ by cron using:
 The systemd directory contains an example unit file with its own
 [README](systemd/README.md).
 
-## Using Ngrok to avoid open ports at the admin end
-
-The instructions above work for a system where the person doing the admin of
-the machine connected to by sshnp is able to run an SSH daemon that's open
-to the Internet. But that's often not practical for many of the same reasons
-why the device can't/won't be reachable directly with an open port. To get
-around this issue it's possible to use the [Ngrok](https://ngrok.com/)
-service as a proxy for the inbound SSH connection.
-
-### Get an Ngrok account
-
-From their [signup page](https://dashboard.ngrok.com/signup)
-
-### Add your SSH public key
-
-From the system you're using for admin:
-
-```
-cat ~/.ssh/id_rsa.pub
-```
-
-Then copy the key and paste it into the `New SSH Key` box on the
-[SSH Public Keys page](https://dashboard.ngrok.com/tunnels/ssh-keys).
-
-### Configure a local SSH server
-
-Such as OpenSSH. It can run on any port, and only needs to be bound to
-localhost. The following example illustrates the use of an SSH server
-bound to port 2222. So the example `/etc/ssh/sshd_config` above becomes:
-
-```
-Port 2222
-#AddressFamily any
-ListenAddress 127.0.0.1
-#ListenAddress ::
-```
-
-### Start a reverse tunnel to Ngrok
-
-It may be useful to do this in a `screen` or `tmux` session
-as another terminal will be needed for `sshnp` later.
-
-```
-ssh -R 0:localhost:2222 tunnel.us.ngrok.com tcp
-```
-
-This will initialise a connection showing something like:
-
-```
-Allocated port 12357 for remote forward to localhost:2222
-
-ngrok (via SSH) (Ctrl+C to quit)
-
-Account     Demo McDemoname (Plan: Free)
-Region      us
-Forwarding  tcp://6.tcp.ngrok.io:12345
-```
-
-### Then invoke sshnp to connect via Ngrok
-
-Command line form:
-
-```
-sshnp -f <@your_manager_atsign> -t <@your_devices_atsign> \
---device <iot_device_name> -h 6.tcp.ngrok.io -p 12345 -l 3456
-```
-
-NB: Ngrok is likely to provide a different tunnel server and port
-each time. So substitute the values from the actual connection for
-`-h 6.tcp.ngrok.io` and `-p 12345`
-
-E.g.
-
-```
-sshnp -f @happyadmin -t @moresecurething \
---device demothing -h 4.tcp.ngrok.io -p 10646 -l 3456
-```
-
-### The tunnel inside a tunnel will now be ready
-
-Connect to it with something like:
-
-```
-ssh -p 3456 -i ~/.ssh/key_for_device.key deviceuser@localhost
-```
-
-Where:
-
-* `-p 3456` corresponds to `-l 3456` from the `sshnp` invocation
-* `-i ~/.ssh/key_for_device.key` is presenting a private key that's trusted
-by the device in its `~/.ssh/authorized_keys`
-* `deviceuser` is the username for the device
-
-### Tunnels in tunnels, an illustration
```

Comment:
It might be useful to keep the ASCII art illustrations, even if we clear out the ngrok references.

Reply:
Will raise a new issue for new diagrams